src/styles/blocks/top-bar.scss

```diff
@@ -0,0 +1,33 @@
+.top-bar {
+  padding-block: 28px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  @include onTablet {
```

Comment:
Use 'padding-top' and 'padding-bottom' instead of 'padding-block'. 'padding-block' is not a valid CSS property.